src/templates/next/components/Infobar/Infobar.tsx

```diff
@@ -0,0 +1,41 @@
+import { InfobarProps } from "~/common"
+import Button from "../Button"
+
+const Infobar = ({
+  sectionIdx,
+  title,
+  description,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+}: InfobarProps) => {
+  return (
+    <section className="py-16 px-5 sm:px-14 lg:py-24">
+      <div className="flex flex-col gap-12 items-center mx-auto text-center lg:max-w-3xl">
+        <div className="flex flex-col gap-7">
+          <h1 className="text-content-default text-4xl leading-tight font-semibold lg:text-5xl lg:leading-tight">
+            {title}
+          </h1>
+          {description && (
+            <p className="text-content-default text-xl">{description}</p>
+          )}
+        </div>
+        <div className="flex flex-row gap-6">
+          {buttonLabel && buttonUrl && (
+            <Button label={buttonLabel} href={buttonUrl} />
+          )}
+          {secondaryButtonLabel && secondaryButtonUrl && (
+            <Button
+              label={secondaryButtonLabel}
+              href={secondaryButtonUrl}
+              colorVariant="white"
+            />
+          )}
+        </div>
+      </div>
```

Comment:
checking to see if these are mobile responsive

Reply:
good catch, forgot about it as it wasn't in design, opting to make it cols on smaller screens for now
[5c1ce43](https://github.com/isomerpages/isomer-components/pull/18/commits/5c1ce430cf949e0d9db82e25071823ed4fcc7a13)

Reply:
Just an aside, how much do we care about a11y? I don't know what's the stance here but just pointing it out